src/main/java/edu/cornell/kfs/concur/rest/jsonObjects/ConcurRequestV4ReportDTO.java

```diff
@@ -0,0 +1,304 @@
+
+    @JsonProperty("requestId")
+    private String requestId;
+
+    @JsonProperty("startDate")
+    private Date startDate;
+
+    @JsonProperty("submitDate")
+    private Date submitDate;
+
+    @JsonProperty("totalApprovedAmount")
+    private ConcurRequestV4AmountDTO totalApprovedAmount;
+
+    @JsonProperty("totalPostedAmount")
+    private ConcurRequestV4AmountDTO totalPostedAmount;
+
+    @JsonProperty("totalRemainingAmount")
+    private ConcurRequestV4AmountDTO totalRemainingAmount;
+
+    @JsonProperty("type")
+    private ConcurRequestV4TypeDTO type;
+
+    public String getHref() {
+        return href;
+    }
+
+    public void setHref(String href) {
+        this.href = href;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public ConcurRequestV4StatusDTO getApprovalStatus() {
+        return approvalStatus;
+    }
+
+    public void setApprovalStatus(ConcurRequestV4StatusDTO approvalStatus) {
+        this.approvalStatus = approvalStatus;
+    }
+
+    public boolean isApproved() {
+        return approved;
+    }
+
+    public void setApproved(boolean approved) {
+        this.approved = approved;
+    }
+
+    public ConcurRequestV4PersonDTO getApprover() {
+        return approver;
+    }
+
+    public void setApprover(ConcurRequestV4PersonDTO approver) {
+        this.approver = approver;
+    }
+
+    public String getBusinessPurpose() {
+        return businessPurpose;
+    }
+
+    public void setBusinessPurpose(String businessPurpose) {
+        this.businessPurpose = businessPurpose;
+    }
+
+    public boolean isCanceledPostApproval() {
+        return canceledPostApproval;
+    }
+
+    public void setCanceledPostApproval(boolean canceledPostApproval) {
+        this.canceledPostApproval = canceledPostApproval;
+    }
+
+    public boolean isClosed() {
+        return closed;
+    }
+
+    public void setClosed(boolean closed) {
+        this.closed = closed;
+    }
+
+    public Date getCreationDate() {
+        return creationDate;
+    }
+
+    public void setCreationDate(Date creationDate) {
+        this.creationDate = creationDate;
+    }
+
+    public ConcurRequestV4CustomItemDTO getChart() {
+        return chart;
+    }
+
+    public void setChart(ConcurRequestV4CustomItemDTO chart) {
+        this.chart = chart;
+    }
+
+    public ConcurRequestV4CustomItemDTO getAccount() {
+        return Account;
+    }
+
+    public void setAccount(ConcurRequestV4CustomItemDTO account) {
+        Account = account;
+    }
+
+    public Date getEndDate() {
+        return endDate;
+    }
+
+    public void setEndDate(Date endDate) {
+        this.endDate = endDate;
+    }
+
+    public boolean isEverSentBack() {
+        return everSentBack;
+    }
+
+    public void setEverSentBack(boolean everSentBack) {
+        this.everSentBack = everSentBack;
+    }
+
+    public boolean isUserReviewed() {
+        return isUserReviewed;
```

Comment:
Another variable that may need to be renamed to just "userReviewed".

Reply:
ok